src/ontology/mondo-edit.obo

```diff
@@ -617527,6 +617527,18 @@ is_a: MONDO:0019118 {source="https://orcid.org/0000-0002-0587-4693"} ! inherited
 intersection_of: MONDO:0019118 {source="https://orcid.org/0000-0002-0587-4693"} ! inherited retinal dystrophy
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/9454 {source="https://orcid.org/0000-0002-0587-4693"} ! PROM1
 
+[Term]
+id: MONDO:1040063
+name: KCNV2-related retinopathy
+def: "Any retinopathy caused by variants in the KCNV2 gene." [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391]
+synonym: "KCNV2-related retinopathy" EXACT CLINGEN_LABEL [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/]
+is_a: MONDO:0019118 ! inherited retinal dystrophy
+intersection_of: MONDO:1040063 {source="https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/", source="https://orcid.org/0000-0001-5935-2391", source="https://orcid.org/0000-0002-0587-4693"} ! KCNV2-related retinopathy
+intersection_of: has_characteristic HP:0000006 {source="https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/", source="https://orcid.org/0000-0001-5935-2391", source="https://orcid.org/0000-0002-0587-4693"} ! Autosomal dominant inheritance
```

Comment:
I would add the inheritance as a SC annotation and not in the definition. 
If you look at the textual definition, you don't mention the inheritance. It just happens that the disease is autosomal dominant.

An example of when I would add the inheritance in the definition is when there is either a AD and AR.